smriprep/workflows/anatomical.py

```diff
@@ -320,14 +321,20 @@ def _check_img(img):
             omp_nthreads=omp_nthreads,
             normalization_quality='precise' if not debug else 'testing')
 
-    # 3. Brain tissue segmentation
+    # 3. Brain tissue segmentation - FAST produces: 0 (bg), 1 (wm), 2 (csf), 3 (gm)
     t1w_dseg = pe.Node(fsl.FAST(segments=True, no_bias=True, probability_maps=True),
                        name='t1w_dseg', mem_gb=3)
 
+    # Change LookUp Table - BIDS wants: 0 (bg), 1 (gm), 2 (wm), 3 (csf)
+    lut_t1w_dseg = pe.Node(niu.Function(function=_apply_bids_lut),
+                           name='lut_t1w_dseg')
+    lut_t1w_dseg.inputs.lut = [0, 3, 1, 2]  # Maps: 0 -> 0, 3 -> 1, 1 -> 2, 2 -> 3.
```

Comment:
we don't expect this to change so let's just make it immutable
```suggestion
    lut_t1w_dseg.inputs.lut = (0, 3, 1, 2)  # Maps: 0 -> 0, 3 -> 1, 1 -> 2, 2 -> 3.
```